src/main/java/manageezpz/logic/commands/EditTaskCommand.java

```diff
@@ -165,7 +173,7 @@ private Task updateEvent(Event currentTask, String desc, String date, String tim
             Time newStartTime = ParserUtil.parseTime(newStartEndTimeStrParts[0]);
             Time newEndTime = ParserUtil.parseTime(newStartEndTimeStrParts[1]);
 
-            if (newEndTime.getParsedTime().compareTo(newStartTime.getParsedTime()) < 0) {
+            if (newEndTime.getParsedTime().compareTo(newStartTime.getParsedTime()) < 1) {
```

Comment:
May i know what the 1 means?

Reply:
This is to factor in the condition that the newEndTime cannot be the same as the newStartTime.

Reply:
Is it possible to abstract it into a method?